crowbar_framework/config/locales/glance/en.yml

```diff
@@ -24,6 +24,7 @@ en:
         keystone_instance: 'Keystone Instance'
         store_header: 'Image Storage'
         store:
+          file_ha_info: 'Please make sure that shared storage is set up on the cluster.'
```

Comment:
I'd rewrite this to: "Shared storage must be used on each node of the High Availability cluster for the file store."